### PR TITLE
Removed space between the short desc if long desc is not present

### PR DESCRIPTION
**Pull request by pavi41**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

**Review by barshat7 — APPROVED** [review]

File: ui.frontend/src/view/FormFieldBase.js [hunk 1]
```diff
@@ -202,11 +202,23 @@ class FormFieldBase extends FormField {
            if (this.getDescription()) {
             ariaDescribedby += `${this.getId()}__longdescription`
           }
-           if (this.getTooltipDiv()) {
-            ariaDescribedby += ` ${this.getId()}__shortdescription`;
+          if (this.getTooltipDiv()) {
```
Comment (rismehta) [inline ui.frontend/src/view/FormFieldBase.js:205]: Can we simplify this code and make it more readable,

`function appendDescription(descriptionType) {
    if (ariaDescribedby) {
        ariaDescribedby += ` ${this.getId()}__${descriptionType}`;
    } else {
        ariaDescribedby = `${this.getId()}__${descriptionType}`;
    }
}

if (this.getTooltipDiv()) {
    appendDescription('shortdescription');
}

if (this.getErrorDiv() && this.getErrorDiv().innerHTML) {
    appendDescription('errormessage');
}`

Comment (rismehta) [reply ui.frontend/src/view/FormFieldBase.js:205]: Please also fix in other places, where there is code duplication

**Review by rismehta — CHANGES_REQUESTED** [review]
check comments

**Review by sakshi-arora1 — APPROVED** [review]

**Comment by barshat7** [comment]
Issue fixed in this PR
https://github.com/adobe/aem-core-forms-components/pull/1228

**Pull request closed without merging**